### PR TITLE
Handling exception while trying describe non-exist instance_id

### PR DESCRIPTION
**Pull request by ilkinmammadzada**
### Description

Exception occurs when try to describe instance to get host information with non-exist instance_id.
Message returns back to queue and process continuous indefinitely.

### Testing Done

Test case was added to check specific scenario.
Test case failed without my change and it was expected.

### CODE REVIEW

**Comment by ilkinmammadzada** [comment]
@jothi7891 did you move strtobool  method for testing?

**Comment by jothi7891** [comment]
@ilkinmammadzada - we decided to vendor the function as that library would soon be deprecated

**Comment by ilkinmammadzada** [comment]
> @ilkinmammadzada - we decided to vendor the function as that library would soon be deprecated

@jothi7891 , it is good idea. 
Additionally, I moved method to more general util file.

File: clusterman/draining/queue.py [hunk 1]
```diff
@@ -291,7 +292,11 @@ def process_warning_queue(self) -> None:
 
 
 def host_from_instance_id(sender: str, receipt_handle: str, instance_id: str,) -> Optional[Host]:
-    instance_data = ec2_describe_instances(instance_ids=[instance_id])
+    try:
+        instance_data = ec2_describe_instances(instance_ids=[instance_id])
+    except ClientError as e:
+        logger.warning(f"Couldn't describe instance: {e}")
```
Comment (jothi7891) [inline clusterman/draining/queue.py]: @ilkinmammadzada - can we get this as `logger.exception` to include the traceback ?

**Review by jothi7891 — APPROVED** [review]

**Pull request merged**